doc/release-process.md

```diff
@@ -50,15 +50,15 @@ and sort them into categories based on labels)
 
 Generate list of authors:
 
-    git log --format='%aN' "$*" | sort -ui | sed -e 's/^/- /'
+    git log --format='- %aN' v(current version, e.g. 0.16.0)..v(new version, e.g. 0.16.1) | sort -fiu
```

Comment:
FWIW the script that I actually use is:
```bash
#!/bin/bash
# Create authors list for changelog
# Example: list_authors.sh v0.9.0rc2..0.9.0
export LC_ALL=C
git log --use-mailmap --format='%aN' "$*" | sort -uif | sed -e 's/^/- /'
```

Reply:
Would you rather I turn this into a script, like you posted, within the ‘devtools’ directory? Being essentially a single line command I didn’t think it worth it, but happy to do so if there’s consensus.

Reply:
@fanquake because there is no shared `mailmap` file in the repo is that actually required?

Reply:
@mitchellcash Fair enough, as this is only an example anyways, lets just leave it as is.